composer.json

```diff
@@ -13,9 +13,9 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "php": ">=5.5.9",
-        "symfony/yaml": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0"
```

Comment:
Weird, the OK are ok on my computer but not in travis, if I revert the Symfony deps, the tests don't pass anymore, can you check ?